### PR TITLE
Fixed Issue #2232: Change Show Log to display all lines of multiline commit comments in the message column

### PR DESCRIPTION
**Pull request by regisbsb**
Added the body of the commit message as per discussion in https://code.google.com/p/tortoisegit/issues/detail?id=2232

### CODE REVIEW

File: src/TortoiseProc/GitLogListBase.cpp [hunk 1]
```diff
@@ -1566,6 +1567,20 @@ CString FindSVNRev(const CString& msg)
 	return _T("");
 }
 
+CString CGitLogListBase::MessageDisplayStr(GitRev * pLogEntry)
+{
+	CString txt = pLogEntry->GetSubject();
+	if (pLogEntry->GetBody().GetLength() > 0)
```
Comment (csware) [inline src/TortoiseProc/GitLogListBase.cpp]: Please use !.IsEmpty()

File: src/TortoiseProc/GitLogListBase.cpp [hunk 2]
```diff
@@ -1566,6 +1567,20 @@ CString FindSVNRev(const CString& msg)
 	return _T("");
 }
 
+CString CGitLogListBase::MessageDisplayStr(GitRev * pLogEntry)
```
Comment (csware) [inline src/TortoiseProc/GitLogListBase.cpp]: Please use `GitRev* pLogEntry`

Comment (regisbsb) [inline src/TortoiseProc/GitLogListBase.cpp]: @csware sorry I did not understand what to fix?

File: src/TortoiseProc/GitLogListBase.cpp (hunk 1 above)
Comment (regisbsb) [inline src/TortoiseProc/GitLogListBase.cpp]: Done!

File: src/TortoiseProc/GitLogListBase.cpp (hunk 2 above)
Comment (csware) [inline src/TortoiseProc/GitLogListBase.cpp]: No space before the asterisk.

Comment (regisbsb) [inline src/TortoiseProc/GitLogListBase.cpp]: Done

**Comment by regisbsb** [comment]
@csware all good now?

**Comment by csware** [comment]
Looks ok. It would be perfect if you could also add a configuration setting for this which defaults to off.

I'm quite busy so it'll take some time until we merge

**Comment by YueLinHo** [comment]
@regisbsb
I see your [post](https://groups.google.com/forum/#!topic/tortoisegit-users/tN41PkKeA0k).
Perhaps you can refer to [this commit](https://github.com/TortoiseGit/TortoiseGit/commit/2e1a785fd3d02cfdd9c53b25668236eaf9d1d81d) for adding the configuration.
Especially the SetDialogs.cpp & SetDialogs.h files.

**Comment by csware** [comment]
Merged. Thanks!

**Comment by regisbsb** [comment]
Merged?

**Comment by csware** [comment]
Yeah, merged.

**Pull request closed without merging**